src/test/java/org/broadinstitute/hellbender/engine/TaggedInputFileArgumentUnitTest.java

```diff
@@ -0,0 +1,98 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.CommandLineParser;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.programgroups.ReadProgramGroup;
+import org.broadinstitute.hellbender.cmdline.programgroups.SparkProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+public class TaggedInputFileArgumentUnitTest extends CommandLineProgramTest {
+
```

Comment:
The unit tests for `TaggedInputFileArgument` need direct testing for parsing of symbolic names and key-value pairs (currently these tests are only in `FeatureInputUnitTest`)